### ISSUE TITLE
'NoneType' object has no attribute 'embed_documents'

### ISSUE THREAD

**Issue by azolfagh** [MaartenGr/BERTopic#911]
Hi Maarten,

I am trying to use the embedding strategy for outlier reduction, and I get the following error. I noticed the same error raises when I choose to use .approximate_distribution using document embedding. I would like to have your insight.

![image](https://user-images.githubusercontent.com/79944982/210803367-70f3395c-c3de-4e3c-a66e-daa90c0ab567.png)

**Comment by MaartenGr** [comment]
In order to use those methods, you will need to have passed an embedding model for BERTopic to use. These methods, to a certain extent, need to calculate embeddings which is not possible without passing an embedding model. This could definitely have been made more clear in both the documentation as in the code. I'll make sure to add some warnings in a future release!

**Comment by azolfagh** [comment]
Thank you for your response. If you mean passing an embedding model as one of the parameters of the topic model, I did so. That's the reason I dont get why this error was raised.
![image](https://user-images.githubusercontent.com/79944982/210829620-7906b57e-a7a3-4141-9f94-5f99fce3d22c.png)

**Comment by MaartenGr** [comment]
Could you share the code for instantiating BERTopic? In the image you shared I can only see you generating embeddings.

**Comment by azolfagh** [comment]
# Sub models
vectorizer_model = CountVectorizer(stop_words="english")
umap_model = UMAP(n_neighbors=15, n_components=5, min_dist=0.0, random_state=42)
ctfidf_model = ClassTfidfTransformer(bm25_weighting=True, reduce_frequent_words=True)
hdbscan_model = HDBSCAN(min_cluster_size=10, metric='euclidean', cluster_selection_method='eom',gen_min_span_tree=True, prediction_data=True)
# BERTopic
topic_model = BERTopic(top_n_words=5, vectorizer_model=vectorizer_model, umap_model =umap_model, hdbscan_model= hdbscan_model, min_topic_size=10, calculate_probabilities=True, verbose=True, ctfidf_model=ctfidf_model)
topics, probs = topic_model.fit_transform(sentences, embeddings)

**Comment by MaartenGr** [comment]
Ah, although you did passed the embeddings, you did not pass the embedding model. You can use the `embedding_model` parameter for that.

**Comment by azolfagh** [comment]
Thank you Maarten, for being such responsive. I will try and let you know.
